database.md

```diff
@@ -78,6 +78,39 @@ Finally, copy the jdbc driver from the database vendor website to the `lib` dire
 
 You will also need to add `DatabaseUsertableQuote="` to your properties file. This is due to the fact that our `user` table is a keyword for postgres.
 
+##### HSQLDB
+
+Unless a different database is configured, the default internal HSQLDB database
+is stored in multiple files in the `$(airsonic.home)/db` directory.
+
+> **NOTE**: It is heavily recommended to operate on a copy of the database (the
+> entire `db` directory) unless write access is needed. Later HSQLDB versions
```

Comment:
What about "It is heavily recommended to operate on a copy of the database (the
 entire `db` directory) unless write access is needed, because modern HSQLDB versions (2+) will **automatically** upgrade the database format, making it unreadable to Airsonic." instead?

Reply:
Why not, done ;)